enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/machines/tx/ReplicatedTransactionFactory.java

```diff
@@ -71,10 +71,6 @@ public static ReplicatedTransaction createImmutableReplicatedTransaction( Transa
             throw new RuntimeException( e );
         }
 
-        /*
```

Comment:
I would keep this comment in, as it's not clear why we do the copy. Saying that we need to trim the zeroes because they are not parseable from the log entry parser and can cause errors is a nice thing to note here.